app/src/renderer/css/components/_slider.css

```diff
@@ -30,10 +30,10 @@
   &::-webkit-slider-thumb {
     -webkit-appearance: none;
     pointer-events: auto;
-    width: 2px;
-    height: 8px;
+    width: 4px;
+    height: 16px;
     border-radius: 8px;
-    transition: all .12s ease;
+    transition: all .2s ease-in-out;
```

Comment:
`.2s` seems too slow for me

Reply:
Did you test it @CodeTheory? It might seem slow, but it doesn't feel like it when you use it, tbh.